templates/sonar_demo.html

```diff
@@ -43,16 +45,18 @@
     // get FFT results and render it on the spectrogram
     fft.event_listeners.push((fft_magnitudes) => {
       if (fft_magnitudes != null) {
-        app.push(fft_magnitudes.slice(395, 415), {z_scale: 5000});
+        //app.push(fft_magnitudes.slice(395, 415), {z_scale: 5000});
+          app.push(fft_magnitudes, {z_scale: 500000});
       }
     });
-
-    const osc = new Tone.Oscillator(19000, 'sine').toDestination();
+    let osc = new Tone.Oscillator(0, 'sine').toDestination();
```

Comment:
- Is there a reason to use `let` instead of `const`?
- Is there a reason to use 0 as the default frequency? 19000 actually wasn't a good default either, but it looks like the [constructor can actually be empty](https://tonejs.github.io/docs/14.7.77/Oscillator.html#baseType).